### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by chcx97**
- converted css to bootstrap

### CODE REVIEW

**Review by annesatran — APPROVED** [review]
I think we should probably remove the console logs, other than that it looks good

File: balto-ui/src/components/RegistrationForm/RegistrationForm.jsx [hunk 1]
```diff
@@ -14,9 +21,11 @@ export default function RegistrationForm() {
                                             password: "",
                                             passwordConfirm: ""
                                           })
+  const [isValidated, setIsValidated] = React.useState(false)
+  const { user, signupUser, error } = useAuthContext()
+  console.log('errors',errors)
```
Comment (LifeBringer) [inline balto-ui/src/components/RegistrationForm/RegistrationForm.jsx]: Replace with console.error

File: balto-ui/src/components/RegistrationForm/RegistrationForm.jsx [hunk 2]
```diff
@@ -14,9 +21,11 @@ export default function RegistrationForm() {
...
+  console.log('form',form)
```
Comment (LifeBringer) [inline balto-ui/src/components/RegistrationForm/RegistrationForm.jsx]: What Annesa said

File: balto-ui/src/components/RegistrationForm/RegistrationForm.jsx [hunk 3]
```diff
@@ -50,123 +58,124 @@ export default function RegistrationForm() {
...
-    if (form.password !== form.passwordConfirm) {
-      setErrors((existingErrors) => ({ ...existingErrors, form: "Passwords do not match" }))
-    }
-
-    if (errors.form) {
+    const registerForm = evt.currentTarget;
+      if (errors.form) {
       console.log("invalid form")
     } else {
       console.log(form)
```
Comment (LifeBringer) [inline balto-ui/src/components/RegistrationForm/RegistrationForm.jsx]: Console.error where applicable

**Review by LifeBringer — APPROVED** [review]
See comments

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See review

**Comment by LifeBringer** [comment]
You can hit 'resolve' once the issues are addressed.

**Review by LifeBringer — APPROVED** [review]

**Pull request merged**